pages/linux/distrobox-create.md

```diff
@@ -0,0 +1,12 @@
+# distrobox-create
+
+> Manual page viewer for distrobox.
```

Comment:
All these descriptions for these pages are the same. The description should describe what the command does.